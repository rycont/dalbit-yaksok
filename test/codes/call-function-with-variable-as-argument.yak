약속, 회전설정 (회전)
    "rotate:" + 회전 반환하기

약속, 시간설정 (시간)
    "time:" + 시간 반환하기

약속, (A) 합 (B)
    A + "<join>" + B 반환하기

약속, (각도)도 회전하기
    회전설정 각도 보여주기

약속, (시간)초 동안 (각도)도 회전하기
    (시간설정 시간) 합 (회전설정 각도) 보여주기

각도 = 45
시간 = 30

(3)초 동안 (90)도 회전하기
3 초 동안 90 도 회전하기
(3)초 동안 90 도 회전하기
3 초 동안 (90)도 회전하기

시간 초 동안 각도 도 회전하기
(시간)초 동안 (각도)도 회전하기
(시간)초 동안 각도 도 회전하기
시간 초 동안 (각도)도 회전하기

(90)도 회전하기
90 도 회전하기
각도 도 회전하기
(각도)도 회전하기