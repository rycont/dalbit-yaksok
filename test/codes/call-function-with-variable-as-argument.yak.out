time:3<join>rotate:90
time:3<join>rotate:90
time:3<join>rotate:90
time:3<join>rotate:90
time:30<join>rotate:45
time:30<join>rotate:45
time:30<join>rotate:45
time:30<join>rotate:45
rotate:90
rotate:90
rotate:45
rotate:45
